### PR TITLE
Alan fix eslint problems

### PR DESCRIPTION
**Pull request by LeeSingChanYau**
This PR fixes mostly the names of variables to camel case and some other lint problems. This PR was to tested and I did not encounter any problems so far, but I will need someone else to make sure there are no issues.

### CODE REVIEW

File: src/controllers/userProfileController.js [hunk 1]
```diff
@@ -3,8 +3,8 @@ const moment = require('moment-timezone');
 const mongoose = require('mongoose');
 const bcrypt = require('bcryptjs');
 
-const userhelper = require('../helpers/userhelper')();
-const TimeEntry = require('../models/timeentry');
+const userHelper = require('../helpers/userHelper')();
+const TimeEntry = require('../models/timeEntry');
```
Comment (Aishwaryak01) [inline src/controllers/userProfileController.js:7]: File name mismatch

File: src/helpers/dashboardhelper.js [hunk 2]
```diff
@@ -1,23 +1,23 @@
 const moment = require('moment-timezone');
 const mongoose = require('mongoose');
 const userProfile = require('../models/userProfile');
-const timeentry = require('../models/timeentry');
-const myTeam = require('../helpers/helperModels/myTeam');
+const timeEntry = require('../models/timeEntry');
```
Comment (Aishwaryak01) [inline src/helpers/dashboardhelper.js:4]: Filename mismatch

File: src/helpers/timeEntryHelper.js [hunk 3]
```diff
@@ -1,13 +1,13 @@
 const moment = require('moment-timezone');
-const timeEntry = require('../models/timeentry');
+const timeEntry = require('../models/timeEntry');
```
Comment (Aishwaryak01) [inline src/helpers/timeEntryHelper.js:2]: File name mismatch

File: src/routes/dashboardRouter.js [hunk 4]
```diff
@@ -7,21 +7,21 @@ const route = function () {
...
+    .get(controller.monthlyData);
 
-  Dashboardrouter.route('/dashboard/weeklydata/:userId/:fromDate/:toDate')
-    .get(controller.weeklydata);
+  Dashboardrouter.route('/dashboard/weeklyData/:userId/:fromDate/:toDate')
+    .get(controller.weeklyData);
 
-  Dashboardrouter.route('/dashboard/leaderboard/:userId')
-    .get(controller.leaderboarddata);
+  Dashboardrouter.route('/dashboard/leaderDoard/:userId')
```
Comment (Aishwaryak01) [inline src/routes/dashboardRouter.js:18]: Typo. D is typed instead of B

File: src/startup/routes.js [hunk 5]
```diff
@@ -1,4 +1,4 @@
-const timeEntry = require('../models/timeentry');
+const timeEntry = require('../models/timeEntry');
```
Comment (Aishwaryak01) [inline src/startup/routes.js:1]: Filename mismatch

File: src/startup/routes.js [hunk 6]
```diff
@@ -18,7 +18,7 @@ const role = require('../models/role');
 const userProfileRouter = require('../routes/userProfileRouter')(userProfile);
 const badgeRouter = require('../routes/badgeRouter')(badge);
 const dashboardRouter = require('../routes/dashboardRouter')();
-const timeEntryRouter = require('../routes/timeentryRouter')(timeEntry);
+const timeEntryRouter = require('../routes/timeEntryRouter')(timeEntry);
```
Comment (Aishwaryak01) [inline src/startup/routes.js:21]: Filename mismatch

**Review by Aishwaryak01 — CHANGES_REQUESTED** [review]
Changes are suggested in the comments. 
Also for src/models/RealTimeTimer.js as the filename is changed, confirm once that it is changed wherever this file is imported.

**Comment by EvianTan** [comment]
Hey @LeeSingChanYau and @Aishwaryak01 @one-community ,

I would suggest also closed this PR for now. Like I said, this PR contains lots of code changes, it's really hard for people to review. I would expect this kind of huge PR to be split into small ones, to reduce the possibility to break our app. Plus, the key points of re-enabling ESLint is not applying changes, instead, which rules should be keep and which rules should be banned. Since Prettier and ESLint have some rules in conflicts with each other.

**Comment by one-community** [comment]
Yes, that makes sense. Aishwarya, is there any way to apply your suggestions here in a different PR so that work isn't wasted?

**Comment by Aishwaryak01** [comment]
The changes are minor but I doubt separating them in different PR as the changes are scattered in multiple files.

**Comment by one-community** [comment]
So do you agree with us just closing this and losing those suggestions?

**Comment by Aishwaryak01** [comment]
If that benefits all, I am happy to close this. Maye the comments can be covered when the PR is split.

**Pull request closed without merging**